target/scripts/startup/setup-stack.sh

```diff
@@ -1742,7 +1742,7 @@ function _setup_user_patches
   if [[ -f ${USER_PATCHES} ]]
   then
     _notify 'tasklog' 'Applying user patches'
-    bash "${USER_PATCHES}"
+    /bin/bash "${USER_PATCHES}"
```

Comment:
Why? To be consequent, we should then also use, e.g. `/bin/sed` across our scripts.

Reply:
Because all our scripts use the shebang

```
#! /bin/bash
```

and the user patches scrips should also be executed this way.

Reply:
`/bin/bash` is also used, when not providing the full path. You can verify this, with `which bash`.

I find it odd, to prefix the full path to some commands, where not necessary. That's whats the $PATH variable is for 😉

Reply:
Right, but I could have also used `/usr/bin/env bash`, but this way, I make it explicit and my intentions clear sich program to call how:)

Reply:
To be clear: I was not talking about the shebang line. I refered to the quoted script line above.